sreejasuresh007-P1.md

```diff
@@ -0,0 +1,182 @@
+# Loading data from csv file into postgresql using python
+
```

Comment:
Add author information like [this](https://github.com/blrgis/blrgis.github.io/pull/1/files#diff-ba4b79cd400f294effc0b7536c5c3ee2R3) because that is the only way for readers to know who wrote the article and how to contact them.

Reply:
Done